The library is libtiff not libTIFF; link with -ltiff not -lTIFF.

--- a/config/params.def
+++ b/config/params.def
@@ -975,7 +975,7 @@ NONSHARED_CCLDFLAGS = NonSharedCCLdFlags
 -lSpatialObjectDEX \
 -lStat \
 -lSurfaceTest \
--lTIFF \
+-ltiff \
 -lTargetDetection-CSU \
 -lTopology \
 -lTraits \
@@ -1023,7 +1023,7 @@ NONSHARED_CCLDFLAGS = NonSharedCCLdFlags
 -lIUE-IF-Region \
 -lDexTestIUE \
 -lIUE-Unit \
--lTIFF
+-ltiff
 #endif
 #else
 #define IueCCLdLibs /**/
@@ -1056,7 +1056,7 @@ NONSHARED_CCLDFLAGS = NonSharedCCLdFlags
 
 #ifndef TiffCCLdLibs
 #ifdef TiffLibDir
-#define TiffCCLdLibs -L$(TIFFLIBDIR) -lTIFF
+#define TiffCCLdLibs -L$(TIFFLIBDIR) -ltiff
 #else
 #define TiffCCLdLibs /**/
 #endif
